crm-project/src/main/java/CRM/controller/facades/SharedContentFacade.java

```diff
@@ -313,7 +314,7 @@ public Response getAllInItem(ObjectsIdsRequest objectsIdsRequest, Class clz) {
     //TODO + DTO list + documentation
     public Response getAllCommentsInSection(ObjectsIdsRequest objectsIdsRequest) {
         try {
-            Validations.validateTwoIds(objectsIdsRequest.getBoardId(), objectsIdsRequest.getSectionId());
+            Validations.validateIDs(Arrays.asList(objectsIdsRequest.getBoardId(), objectsIdsRequest.getSectionId()));
```

Comment:
You don't need to put it in an array, just send it the same way as before.
Scroll down to see how you accept them in the "validateIDs" function